runtime/util/util.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2020 The Flux authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
```

Comment:
Maybe `transform`? Hard to make suggestions at this moment because there is little that falls into the same category.

Reply:
Ya I'm horrible at naming things. I've changed it to be `transform`. Don't be shy if you want something else though. I don't mind changing it, anytime! 😁